config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-periodics-main.yaml

```diff
@@ -0,0 +1,34 @@
+periodics:
+- name: periodic-cluster-api-provider-vsphere-coverage
+  interval: 24h
+  decorate: true
+  path_alias: "sigs.k8s.io/cluster-api-provider-vsphere"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-vsphere
+      base_ref: main
+      path_alias: "sigs.k8s.io/cluster-api-provider-vsphere"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220124-681f3531ec-master
```

Comment:
Could you use the same image version that are used by the other periodic jobs so that it is consistent for all the periodic jobs?